### ISSUE TITLE
ADE Finding: User menu button missing aria attributes & violates label in name

### ISSUE THREAD

**Issue by zslocum1** [department-of-veterans-affairs/va.gov-team#131691]
## Background/Context

This ticket is to address two ADE findings that will not require troubleshooting with a screen reader. Ideally this would ensure that is works on desktop w/ Voice Control and Voiceover.

The user menu button is not announced as expected and is missing needed aria attributes. Rec: Because the user menu button causes an additional popup to be displayed, the button should have aria-haspopup="true". This will cause the button to be announced as "menu popup collapsed" as opposed to "toggle menu", which VoiceOver users will expect.

The accessible name is determined by the content of the button: the screen reader text on the icon ("toggle menu") and the button label ("[First] [Last]"). Remove the screen reader text from the icon, which is purely decorative, to ensure the button can be activated by the text that is available visually.

VA Severity: 3
Estimated 508 Severity: 2

<img width="1269" height="593" alt="Image" src="https://github.com/user-attachments/assets/727451f6-d8d0-41ee-9b4f-014f11fa64ec" />

## Expected vs. Actual
**Expected**: 
The accessible name for the user menu button is "toggle menu [First] [Last]". I would expect to hear "[First] [Last], menu popup collapsed, button."

**Actual**: 
The only visible text is "[First] [Last]". VoiceOver announces the button as "toggle menu [First] [Last], Collapsed, button," but "toggle menu" is coming from the screen reader text on the icon and is not how buttons are typically announced. Voice Control users must say "Click toggle menu [First] [Last] to interact with the element, which is impossible to know by looking at it.

**Comment by zslocum1** [comment]
Confirmed both the aria attributes have been updated and user menu button

<img width="595" height="98" alt="Image" src="https://github.com/user-attachments/assets/fd81aec8-ae10-460d-a6b4-fb4c865477bd" />
<img width="2232" height="850" alt="Image" src="https://github.com/user-attachments/assets/512e422d-676b-4524-89b5-84b5ee2c1383" />
